spec/factories/event_users.rb

```diff
@@ -0,0 +1,9 @@
+# frozen_string_literal: true
+
+FactoryBot.define do
+  factory :event_user do
+    event_id { Faker::Number.within(range: 1..10) }
```

Comment:
You should create event and attach it to event_user
You can do it very simple
```suggestion
    event
```

Reply:
Created `event` and `user` and attached it to `event_user` factory